src/providers/setting-service.ts

```diff
@@ -73,7 +73,7 @@ export class SettingService {
   }
 
   isShowRestUrlField(): Observable<boolean> {
-    return this.readKey(this.isShowRestUrlFieldKey).map(val => val == null ? this.isShowRestUrlFieldDefault : val);
+    return this.readKey(this.isShowRestUrlFieldKey).map(val => val === undefined ? this.isShowRestUrlFieldDefault : val);
```

Comment:
Leads to server field not being displayed per default. Should be changed back to `null`.